dbt_platform_helper/templates/pipelines/codebase/overrides/stack.ts

```diff
@@ -108,7 +108,7 @@ export class TransformedStack extends cdk.Stack {
                 type: 'LINUX_CONTAINER',
                 computeType: 'BUILD_GENERAL1_SMALL',
                 privilegedMode: true,
-                image: 'public.ecr.aws/uktrade/ci-image-builder:tag-latest',
+                image: 'public.ecr.aws/uktrade/ci-image-builder',
```

Comment:
This should not have gone away.
```suggestion
                image: 'public.ecr.aws/uktrade/ci-image-builder:tag-latest',
```

I must release https://github.com/uktrade/platform-tools/pull/544